### DISCUSSION TITLE
Request: Description on meo.pt guide

### DISCUSSION THREAD

**Discussion by thomraider12** [iptv-org/community#1148] [Show and tell]
Hello, I, and many people from my country, would want to have description on the EPG source of "MEO.pt". Maybe this code that me and ChatGPT made would work.

You can find it [here](https://raw.githubusercontent.com/thomraider12/epg/sites/meo.pt/meo.pt.config.js)

If added, much appreciated.
Thanks.

**Comment by freearhey** [comment]
> 404: Not Found

https://raw.githubusercontent.com/thomraider12/epg/sites/meo.pt/meo.pt.config.js

**Reply by thomraider12** [reply]
I already closed it, I think it didn't work, though, that would be a great suggestion.
